### PR TITLE
Submission templates

### PR DESCRIPTION
**Pull request by drio18**
This PR adds a command to generate submission worksheet templates based on the schemas on the chosen environment. The template can be generated as an Excel workbook or can be updated to the designated google sheet.

Up-to-date google sheet created by the new command with the updated schemas can be found [here](https://docs.google.com/spreadsheets/d/1ipCSFViDVcwS-5YjFpj1z6CnDdWVjMO-FFJBuUaKF1s/edit?usp=sharing).

Additionally, many small schema updates, predominantly for various property descriptions, are included based on recent feedback.

### CODE REVIEW

File: src/encoded/schemas/death_circumstances.json [hunk 1]
```diff
@@ -85,7 +86,7 @@
         },
         "cause_of_death_immediate_interval": {
             "title": "Cause Of Death Immediate Interval",
-            "description": "Interval of time from immediate cause of death to death in minutes",
+            "description": "Interval of time from immediate cause of death to death (minutes)",
```
Comment (sarahgonicholson) [inline src/encoded/schemas/death_circumstances.json:89]: In the spreadsheet the TPC commented they would prefer units of "hours" for cause_of_death_immediate_interval, death_pronounced_interval, and ventilator_time.

**Review by sarahgonicholson — APPROVED** [review]

**Review by wfeng19 — APPROVED** [review]
All of the changes look good to me

File: src/encoded/commands/write_submission_spreadsheets.py [hunk 2]
```diff
@@ -0,0 +1,836 @@
...
+        spreadsheet_client = get_google_sheet_client(args.google)
+        update_google_sheets(spreadsheet_client, request_handler)
+    elif args.all:
+        log.info("Writing all submission spreadsheets")
+        write_all_spreadsheets(
+            args.output,
+            request_handler,
+            workbook=args.workbook,
+            separate_comments=args.separate,
+            google=args.google,
```
Comment (sarahgonicholson) [inline src/encoded/commands/write_submission_spreadsheets.py:770]: I think line 770 needs to be removed. Running `write_submission_spreadsheets -output ~/Downloads --env staging --workbook --all` gave an error that google wasn't an acceptable argument and commenting out this line fixed the issue.

Comment (drio18) [reply src/encoded/commands/write_submission_spreadsheets.py:770]: Ah, thanks, that was a leftover from an earlier stage of development.

**Pull request merged**